spec/common/importers/firefoxCsvImporter.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { FirefoxCsvImporter as Importer } from '../../../src/importers/firefoxCsvImporter';
+
+import { CipherView } from '../../../src/models/view/cipherView';
+import { LoginView } from '../../../src/models/view/loginView';
+import { LoginUriView } from '../../../src/models/view/loginUriView';
+
+import { Utils } from '../../../src/misc/utils';
+
+import { FieldType } from '../../../src/enums';
+
+if (Utils.isNode) {
+    // Polyfills
+    // tslint:disable-next-line
+    const jsdom: any = require('jsdom');
+    (global as any).DOMParser = new jsdom.JSDOM().window.DOMParser;
+}
+
+const CipherData = [
```

Comment:
I know this matches the lastpass importer tests, but to clean up these test files, I'd prefer the csv strings exported from a testData file similar to SafeInCloud and 1Password export data.